io-engine/src/core/snapshot.rs

```diff
@@ -320,6 +320,9 @@ pub trait SnapshotOps {
         &self,
         parent: Option<&Lvol>,
     ) -> Option<VolumeSnapshotDescriptor>;
+
+    /// Returns a boolean indicating if the lvol is a clone from snapshot.
+    fn is_snapshot_clone(&self) -> bool;
```

Comment:
Shall we also somehow provide the snapshot this clone was created from ? I presume for user it's also important to be able to answer the question "what's the parent of this clone ?".
Also, this clone-related function is part of 'pub trait SnapshotOps',  which is snapshot-related.
Isn't it better to have 'is_snapshot_clone()' be part of Lvol or so ?

Reply:
1. Currently as part of list_clone gRPC, We are returning snapshot uuid from which clone is created. Do We need put it in any other gRPC response ?
2. For 2nd point, I will do the change. I was also little confuse if to put it in snapshot category or lvol. You are correct, as this will work for replica also, I will keep part of lvol trait.

Reply:
> 1. Currently as part of list_clone gRPC, We are returning snapshot uuid from which clone is created. Do We need put it in any other gRPC response ?
> 2. For 2nd point, I will do the change. I was also little confuse if to put it in snapshot category or lvol. You are correct, as this will work for replica also, I will keep part of lvol trait.

Got it for point: 1, I will optimize the is_snapshot_clone() API to optimize the query used in list_clone gRPC.